sonar-db/src/main/java/org/sonar/db/permission/PermissionQuery.java

```diff
@@ -71,6 +71,8 @@ public boolean withPermissionOnly() {
     return withPermissionOnly;
   }
 
+  // TODO remove it, it should not be in the query, but set as a separate parameter
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Complete the task associated to this TODO comment. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3AS1135)